tests/native-lib/pass/scalar_arguments.rs

```diff
@@ -1,4 +1,5 @@
-//@only-target-linux
+//@ignore-target-windows
```

Comment:
```suggestion
// Only works on Unix targets
//@ignore-target-windows
```
same for the other file